src/main/java/com/squareup/javapoet/Util.java

```diff
@@ -82,4 +154,23 @@ public static String join(String separator, List<String> parts) {
     result.addAll(b);
     return result;
   }
+
+  public static Set<Modifier> getMissingImplicitModifiers(Set<Modifier> modifiers,
```

Comment:
:+1: Was going to say something similar (and if you look at the mercurial commit where private methods were added you see they do similar things).